experiment/measurer/test_run_coverage.py

```diff
@@ -13,8 +13,6 @@
 # limitations under the License.
 """Tests for run_crashes.py."""
 
-# pylint: disable=no-self-use
```

Comment:
We may need to keep these for now until we update pylintrc

Reply:
> Thanks for this. i think we will need to discuss this offline. I have some concerns. Mainly, why update .pylintrc?

I updated `.pylintrc` mainly because I updated `pylint`, which is because we updated `python`.
This `.pylintrc` is automatically generated with the latest version of `pylint`, which I presume contains many new settings (e.g., some default settings may have been changed to optional).
I am also working on manually merging some of our old changes to here, e.g. `ignore=alembic`

Happy to discuss more on this, as I am not certain which settings should be kept.

Reply:
Just come across another reason to update `.pylintrc`: I found 35 deprecated and 5 unknown `pylint` messages in our old `.pylintrc` that cause errors with the new `pylint`.